### PR TITLE
Add Options

### PR DESCRIPTION
**Pull request by JavierAybar**
**_The following options should be available:_**
- List of games
- List all authors (e.g. 'Stephen King')
- Add a game

### - **_Changes related with `main.rb` and **app.rb** are just for test my options(I forgot to delete before committing)_**

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -2,7 +2,8 @@
 require_relative 'app/app'
 
 def main
-  App.new
+  app = App.new
+  app.init
 end
```
Comment (diarisdiakite) [inline main.rb:7]: Please, would just like to know, is "init" a native method or is it implemented somewhere in the code?

**Review by diarisdiakite — APPROVED** [review]
Well done @JavierAybar 
- Code well strucured
- No linters errors

Please find one slight question in the codebase. 
Otherwise, everything seems fine until we are able to test the app.

File: main.rb (hunk 1 above)
Comment (JavierAybar) [reply main.rb:7]: Hello @diarisdiakite init isn't a native method, I created it to check if my option works as I told in the PR:
### - Changes related with main.rb and app.rb are just for test my options(I forgot to delete before committing)
I just forgot to delete those changes, we can delete them if necessary.

**Comment by JavierAybar** [comment]
Thanks to review my PR @diarisdiakite

**Pull request merged**